src/actions/ui.js

```diff
@@ -100,9 +100,10 @@ export const tClearUi = () => (dispatch, getState) => {
     )
 }
 
-export const acSetUiActiveModalDialog = (value) => ({
+export const acSetUiActiveModalDialog = (value, metadata) => ({
```

Comment:
Unsure about this pattern of having `metadata` as an additional prop in the request here, especially since its not entirely clear what the metadata is referring to (renaming the action to `acSetUiActiveDimensionModal` might help with that). Is it preferred to use a consolidated action like this or should the consumer just do a separate "add metadata" call instead? @janhenrikoverland

Reply:
> or should the consumer just do a separate "add metadata" call instead?

If this is the preferred way, we should probably remove the redux middleware.

Reply:
No I like the redux middleware, just want it to be clear what the metadata is referring to. So best would be to rename the action to `acSetUiActiveDimensionModal` so its obvious that its a "dimension" that is open (hence said dimension can have metadata related to it, which is passed as an optional prop). Also, whats a "ModalDialog" anyway? 😄

Reply:
one could also argue that "active" is too loose. "open" makes more sense (as thats what modals do, open and close, not activate and deactivate)... so `acSetUiOpenDimensionModal` 🎉